torch/nn/parallel/data_parallel.py

```diff
@@ -17,6 +19,8 @@
 from torch.nn.parallel.replicate import replicate
 from torch.nn.parallel.scatter_gather import gather, scatter_kwargs
 
+P = ParamSpec("P")
```

Comment:
Also probably should be underscore since it's not supposed to be a public TypeVar